kyuubi-server/src/test/scala/org/apache/kyuubi/events/handler/ServerJsonLoggingEventHandlerSuite.scala

```diff
@@ -197,6 +198,7 @@ class ServerJsonLoggingEventHandlerSuite extends WithKyuubiServer with HiveJDBCT
     server.initialize(conf)
     server.start()
     server.stop()
+    ServerEventHandlerRegister.registerEventLoggers(conf)
```

Comment:
According to @wForget 's advice, stopping the server in line 200 will also clear the event handlers in the singleton of EventBus. Re-register the event loggers here helps for continuing sending to existed event logger in server for the whole testsuite.

Reply:
Let's separate the closing event handler to this PR https://github.com/apache/kyuubi/pull/4777.